deployment/src/main/java/io/quarkiverse/qute/web/deployment/QuteWebTemplateBuildItem.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkiverse.qute.web.deployment;
+
+import static io.quarkiverse.qute.web.runtime.PathUtils.removeLeadingSlash;
+import static io.quarkiverse.qute.web.runtime.PathUtils.removeTrailingSlash;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class QuteWebTemplateBuildItem extends MultiBuildItem {
+
+    /**
+     * templateId is used as path if link is null
+     */
+    private final String templateId;
```

Comment:
I think that this field should still be named `path` because it's the "real" path to the template file, or?

`templateId` in general is used as a key in the `Engine` and it could be anything...

Reply:
done